README.md

```diff
@@ -1,80 +1,106 @@
-
-5. Now that the repository has been forked and the GitHub pages settings are configured, you can commit and push your inputs using the GitHub web console.
+2. Create a new repository and give it a name.
+![The create new repository image shows the GitHub web console for creating a new repository.](docs/images/create_new_repo.png "Create Repository")
+
+3. Create a new file with extension `.yaml` (i.e. an YAML file) under the `.github/workflows` folder and give it a name (e.g. action.yaml). Then add the below content into it and commit the file.
+
+> NOTE: Read the comments in below content to learn more about the supported configurations.
+
+```YAML
+# Name of the workflow
+name: csv-to-csvw example
+# How the workflow should be triggered.
+on:
+  push:
+    branches:
+      - main # This workflow is triggered when committed to the main branch.
+# Workflow action
+jobs:
+  generate_csvw_from_csv_upload:
+    name: Generate CSV-W from csv upload
+    runs-on: ubuntu-latest
+    steps:
+      - name: csv-to-csvw action
+        # csv-to-csvw action path. The latest version is always available at https://github.com/marketplace/actions/csv-to-csvw-action
+        uses: GSS-Cogs/csv-to-csvw-action@v0.0.29-alpha
+        # Configurations
+        with:
+          # Boolean indicating whether the outputs (i.e. out folder) should be committed to the current branch (default is true).
+          commit-outputs-to-branch: true
+          # Boolean indicating whether the outputs (i.e. out folder) should be published to the gh-pages branch and GitHub Pages (default is true).
+          publish-to-gh-pages: true
+```
+![The create workflow file image shows creating and commiting a new action.yaml file.](docs/images/create_workflow_file.png "Create Workflow File")
+
+4. Create a new branch called `gh-pages`. This branch will be used to publish the outputs to GitHub Pages. Then go to the repository's settings and set the branch for GitHub Pages - under the `Source` section, set the `Branch` to `gh-pages` and set the folder location to `/(root)`. Also, keep a note of the URL at which your GitHub Pages site is published at.
+![GitHub Pages setting shows the branch set for GitHub Pages. The URL at which the GitHub Pages site is published at is also displayed in this setting.](docs/images/github_pages_setting.png "GitHub Pages Setting")
+
+5. Now that the repository is created and the GitHub Pages setting is configured, you can commit and push your inputs using the GitHub web console.
 ![The commit inputs image shows the Upload Files option provided in the GitHub web console.](docs/images/commit_files.png "Commit Inputs")
 
 6. Once the inputs have been committed, the action will automatically [run](#key-steps-performed-by-the-action). To see the progress of the action, go to the `Actions` section in the GitHub web console.
```

Comment:
I think the steps 6+ should be part of the `Build Process` section

Reply:
I have restructured this a bit. Now there are only 5 steps the user needs to perform. Please let me know whether you are happy with how this section looks now.